### PR TITLE
HTML boilerplate also added classes and Id's

### PR DESCRIPTION
**Pull request by MelanieRBaker**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	<p>Messages:</p>
+	<div id="message-box">
+
+	</div>
+	<script src="dom.js"></script>
+	<script src="chatty-array.js"></script>
+	<script src="chatty-dom.js"></script>
+	<script src="chatty-json.js"></script>
+</body>
+</html>
```
Comment (aoswalt) [inline index.html]: missing eof newline

**Comment by aoswalt** [comment]
:+1:

**Comment by carter-harris** [comment]
👍

**Comment by castle-c** [comment]
:octocat:

**Pull request merged**